src/app/public/public_api.ts

```diff
@@ -1,6 +1,9 @@
 export * from './modules/sectioned-form/sectioned-form.module';
 export * from './modules/sectioned-form/sectioned-form.service';
 
+export * from './modules/tabs/tab-index';
+export * from './modules/tabs/tabset-buttons-display-mode';
```

Comment:
I'm not seeing where `SkyTabsetButtonsDisplayMode` is used by consumers. Is that supposed to be part of the public API?

Reply:
Yep! It's similar to `SkyTabsetButtonsDisplayMode`.